robot-command/src/main/java/org/obolibrary/robot/RenameCommand.java

```diff
@@ -0,0 +1,243 @@
+   * extension (CSV = comma, TSV/TXT = tab).
+   *
+   * @param fileName name of file to get table separator for
+   * @return character separator, either tab or comma
+   * @throws IOException if file is not TSV/TXT or CSV
+   */
+  private static char getSeparator(String fileName) throws IOException {
+    if (fileName.endsWith(".tsv") || fileName.endsWith(".txt")) {
+      return '\t';
+    } else if (fileName.endsWith(".csv")) {
+      return ',';
+    }
+    throw new IOException(String.format(fileFormatError, fileName));
+  }
+
+  /**
+   * Given a file containing the mappings table and a character to separate cells, return the map of
+   * old values (left) to new values (right). Partial mappings can allow duplicate values, but full
+   * mappings should provide a warning and allow user to kill the process if there are duplicate
+   * values.
+   *
+   * @param mappingsFile File to parse mappings from
+   * @param separator character to separate cells in mapping file
+   * @param warnDuplicateValues boolean indicating if duplicate map values should provoke a warning
+   * @return map of old values -> new values
+   * @throws IOException if the file does not exist, if each row does not have exactly two cells, or
+   *     if there are multiple mappings for one key (old value)
+   */
+  private static Map<String, String> parseTableMappings(
+      File mappingsFile, char separator, boolean warnDuplicateValues) throws IOException {
+    Map<String, String> mappings = new HashMap<>();
+
+    if (!mappingsFile.exists()) {
+      throw new IOException(String.format(missingFileError, mappingsFile.getPath()));
+    }
+
+    try (CSVReader reader =
+        new CSVReaderBuilder(new FileReader(mappingsFile))
+            .withCSVParser(new CSVParserBuilder().withSeparator(separator).build())
+            .build()) {
+      int lineNum = 1;
+      // Remove the headers
+      reader.readNext();
+      for (String[] nextLine : reader) {
+        lineNum++;
+        if (nextLine.length != 2) {
+          throw new IOException(String.format(columnCountError, lineNum, mappingsFile.getPath()));
+        }
+        if (mappings.containsKey(nextLine[0])) {
+          throw new IOException(
+              String.format(duplicateMappingError, lineNum, mappingsFile.getPath(), nextLine[0]));
+        }
+        if (mappings.containsValue(nextLine[1]) && warnDuplicateValues) {
+          Scanner s = new Scanner(System.in);
+          System.out.println(
+              String.format(
+                  "---------- WARNING ----------\n"
+                      + "Line %d in file '%s' contains a duplicate value ('%s').\n"
+                      + "This will rename two separate entities to have the same IRI, resulting in a merge."
+                      + "\nDo you wish to continue? [y/N]",
+                  lineNum, mappingsFile.getPath(), nextLine[1]));
+          String cont = s.nextLine();
```

Comment:
I given this some thought, and I've decided that promtping for user input is a bad idea. ROBOT is meant to be used for automation. None of our other commands are interactive. `make` will get stuck forever if it hits this warning, which will surprise and annoy our users.

I think the better alternatives are:

1. just log a warning
2. add a `--fail-on-duplicates true/false` option